ppml/utils/kms-utils/entrypoint.sh

```diff
@@ -0,0 +1,211 @@
+# set -x
```

Comment:
should provide 3 basic utilities
1. enroll
2. generate keys including primary key/ data key
3. encrypt including encrypt and split_and_encrypt
4. decrypt